packages/applications-service-api/prisma/schema.prisma

```diff
@@ -104,3 +104,18 @@ model Document {
 
   @@index([caseRef])
 }
+
+model ProjectUpdate {
+  id                   Int       @unique @default(autoincrement())
+  projectUpdateId      Int       @unique
+  caseReference        String
+  updateDate           String?
```

Comment:
Looking at BO this is date the update is published.  We are looking to display the latest published update and our API will likely sort the response by this field.  I wonder if we should be enforcing it as mandatory.  Perhaps we can note it and do it later.

Reply:
Confirmed it will be present when we receive the message. Will make it non-optional in our schema here